packages/contracts/src/2.0.0/protocol/AssetProxy/ERC721Proxy.sol

```diff
@@ -83,34 +83,26 @@ contract ERC721Proxy is
                 // WARNING: The ABIv2 specification allows additional padding between
                 //          the Params and Data section. This will result in a larger
                 //          offset to assetData.
-                
```

Comment:
Question: Do you think we should change the ID of this proxy?

It is currently:
```
bytes4 constant internal PROXY_ID = bytes4(keccak256("ERC721Token(address,uint256,bytes)"));
```

We could change this to:
```
bytes4 constant internal PROXY_ID = bytes4(keccak256("ERC721Token(address,uint256)"));
```

I've left it unchanged for now so that forwarding contracts can include the additional bytes, if they'd like to implement callbacks.

Reply:
Yeah, I suppose in both cases the extra data would just be displayed as an unstructured byte array. I'll make the update.

Reply:
* Updated